src/Sarif.Sarifer/SariferOption.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+using Microsoft.VisualStudio.Shell;
+
+using Task = System.Threading.Tasks.Task;
+
+namespace Microsoft.CodeAnalysis.Sarif.Sarifer
+{
+    internal class SariferOption
+    {
+        private readonly AsyncPackage package;
+
+        private SariferExtensionOptionPage optionPage;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="SariferOption"/> class.
+        /// Get visual studio option values
+        /// </summary>
+        /// <param name="package">Owner package, not null.</param>
+        private SariferOption(AsyncPackage package)
+        {
+            this.package = package ?? throw new ArgumentNullException(nameof(package));
+        }
+
+        public SariferExtensionOptionPage OptionPage
+        {
+            get
+            {
+                this.optionPage ??=
+                    this.optionPage = (SariferExtensionOptionPage)this.package.GetDialogPage(typeof(SariferExtensionOptionPage));
```

Comment:
should this be:
this.optionPage ??= (SariferExtensionOptionPage)this.package.GetDialogPage(typeof(SariferExtensionOptionPage));

Reply:
done

---
In reply to: [564953835](https://github.com/microsoft/sarif-visualstudio-extension/pull/354#discussion_r564953835) [](ancestors = 564953835)